docs/build.sh

```diff
@@ -8,14 +8,27 @@ source ../ci/env.sh
 
 : "${rust_stable_docker_image:=}" # Pacify shellcheck
 
+# Fixate master branch as only one to download translations & get its ref
+ONLY_TRANSLATE_ON='master'
+MASTER_REF="$(git rev-parse "$ONLY_TRANSLATE_ON")}"
+# Get current ref
+CURRENT_REF="$(cat ../.git/HEAD)"
+
+# Synchronize translations with Crowdin only on master branch (ref)
+if [[ "$CURRENT_REF" = "$MASTER_REF" ]]; then
+  echo "Downloading & updating translations..."
+  npm run crowdin:download
+  npm run crowdin:upload
+fi
+
 # shellcheck source=ci/rust-version.sh
 source ../ci/rust-version.sh
 ../ci/docker-run.sh "$rust_stable_docker_image" docs/build-cli-usage.sh
 ../ci/docker-run.sh "$rust_stable_docker_image" docs/convert-ascii-to-svg.sh
 ./set-solana-release-tag.sh
 
 # Build from /src into /build
-npm run build
+#npm run build
```

Comment:
Lets kill the block instead of committing commented code

Reply:
Ups, that was a remainder from testing it % ). Uncommented again.